packages/clerk-js/src/utils/completeSignUpFlow.ts

```diff
@@ -22,9 +22,9 @@ export const completeSignUpFlow = ({
   if (signUp.status === 'complete') {
     return handleComplete && handleComplete();
   } else if (signUp.status === 'missing_requirements') {
-    if (signUp.missingFields.some(mf => mf === 'saml')) {
+    if (signUp.missingFields.some(mf => mf === 'saml' || mf === 'enterprise_sso')) {
       return signUp.authenticateWithRedirect({
-        strategy: 'saml',
+        strategy: 'enterprise_sso',
```

Comment:
❓For backward compatibility, shouldn't we set `strategy` equivalent to the missing field? Essentially, `saml` or `enterprise_sso`
```suggestion
        strategy: mf,
```

Reply:
Hmm, wanted to remove the usage of saml from the code base, a similar change was made to [sign in as well](https://github.com/clerk/javascript/blob/nikpolik%2Fuser-939-add-support-for-enterprise_sso-strategy-in-clerkjs/packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx#L285-L288). 

The `enterprise_sso` strategy can be used regardless of the version so it seems ok? 🤔